evil-collection-debbugs.el

```diff
@@ -66,6 +66,11 @@
     ;; refresh
     "gr" 'debbugs-gnu-rescan
 
+    "g?" 'debbugs-gnu-manual
+    "gt" 'debbugs-gnu-view-bug-triage
+    "g#" 'debbugs-gnu-bugs
+    "S" 'debbugs-gnu-search
```

Comment:
Much better indeed.  I think I'd move this line next to "s" though, to highlight the link between the two.

Reply:
Will do.